### ISSUE TITLE
Possibility of a version for non-jailbroken devices?

### ISSUE THREAD

**Issue by PencilNotPen** [haoict/twitternoads#21]
Hi, it is possible to inject tweaks into .ipa files and sideload them to non-jailbroken devices, but only if they only tweak the app, and do not require system dependencies.

Would it be possible to make a version of this tweak (or in fact any of your tweaks, like for Facebook and messenger as well), which works when injected into stock .ipas? I think it is only the dependencies, or perhaps the inject into Settings with PreferenceLoader, which does not work. A version without settings (automatic ad-block without settings), or a version with in-app settings, and without external dependencies, should work.

Do you think this is in any way possible for any of your adblocking tweaks?

Thank you very much for your hard work and excellent tweaks!

**Comment by dobby1970** [comment]
yeah I have the same question! it would be so nice, no tweak for blocking twitter ads is working when injected and sideloaded
